src/client.py

```diff
@@ -315,31 +309,34 @@ def vbs_block_process(process, popup, message, title, timeout, type):
                  "Do" + "\n" + "Set objLatestProcess = colMonitoredProcesses.NextEvent\n" + \
                  f"If LCase(objLatestProcess.TargetInstance.Name) = \"{process}\" Then\n" + \
                  "objLatestProcess.TargetInstance.Terminate\n"
-    if popup == "True":  # if showing a message
-        strVBSCode += f'objWshShl.Popup "{message}",{timeout}, "{title}",{type}\n'
+    if popup:  # if showing a message
+        strVBSCode += f'objWshShl.Popup "{popup[0]}", {popup[1]}, "{popup[2]}", {popup[3]}\n'
```

Comment:
there should be quotes around the title but not the 3rd argument.

Reply:
I'm not sure what you mean, the message and title have quotes and the others don't

Reply:
Right now this is the code in the VBS file

`objWshShl.Popup "Task Manager has been disabled by your administrator", Task Manager, "3", 16`

the problem is the arguments are not in the right order

popup = ["Task Manager has been disabled by your administrator", "Task Manager", "3", "16"]